index.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <title>My CV</title>
+</head>
+
+<header>
+    <h1>
+        Santiago Urrego
+    </h1>
+    <img src="perfil.jpg" alt="Foto de perfil" title="Urregod"width="150" height="192">
+</header>
+
+<body>  
+    <h2>Educación</h2>
+    <ul>
+        <li>Universidad de La Sabana: 2023-Actualidad</li>
+        <li>Colegio Colombo Hispano: 2018-2021</li>
+        <li>Colegio Canapro: 2014-2017</li>
+        <li>Instituto Infantil y Juvenil: 2010-2013</li>
+    </ul>
```

Comment:
```suggestion
<body>  

    <header>
        <h1>
            Santiago Urrego
        </h1>
        <img src="perfil.jpg" alt="Foto de perfil" title="Urregod"width="150" height="192">
    </header>

    <h2>Educación</h2>
    <ul>
        <li>Universidad de La Sabana: 2023-Actualidad</li>
        <li>Colegio Colombo Hispano: 2018-2021</li>
        <li>Colegio Canapro: 2014-2017</li>
        <li>Instituto Infantil y Juvenil: 2010-2013</li>
    </ul>
```

El header va dentro del **body**